src/components/trifold/groupSuggestionsSidebar.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import groups from "../../data/groups/groupsData";
+import GroupType from "../../modules/groupType";
+import GroupSuggestionsItem from "./groupSuggestionsItem";
+
+const GroupSuggestionsSidebar = () => {
+  return (
+    <div className="list-group mb-1">
+      {
+        groups.map((group : GroupType) => {
+        return (
+          <GroupSuggestionsItem {...group} />                

```

Comment:
seeing an warning in the console related to this.... "Warning: Each child in a list should have a unique "key" prop."

pass in key = {group._id} to prevent it!